engine/execution/computation/computer/computer.go

```diff
@@ -449,39 +441,33 @@ func (e *blockComputer) executeTransaction(
 
 	memAllocAfter := debug.GetHeapAllocsBytes()
 
-	lg := e.log.With().
-		Str("tx_id", txID.String()).
-		Str("block_id", blockIdStr).
-		Str("traceID", traceID).
+	logger = logger.With().
 		Uint64("computation_used", tx.ComputationUsed).
 		Uint64("memory_used", tx.MemoryEstimate).
-		Uint64("memAlloc", memAllocAfter-memAllocBefore).
-		Int64("timeSpentInMS", time.Since(startedAt).Milliseconds()).
+		Uint64("mem_alloc", memAllocAfter-memAllocBefore).
+		Int64("time_spent_in_ms", time.Since(startedAt).Milliseconds()).
```

Comment:
you may consider renaming other two instead: `timeSpentInMS` is quite common in the repo

Reply:
all other attributes (block_id, tx_index, etc.) are logged using snake case.  it's better to be consistent within the same log line rather consistently inconsistent